docs/host/host.md

```diff
@@ -80,3 +80,81 @@ If you are testing Harvester in a QEMU environment, you'll need to use QEMU v6.0
+3. Select the **Ksmtuned** tab and **Run Strategy** in Run.
+4. (Optional)You can change **Threshold Coefficient** to match your expectations.
+
+![Edit Ksmtuned](/img/v1.1/host/edit-ksmtuned.png)
+
+5. Click **Save** to update.
+6. Wait at least 2 minutes and check the **Statistics**, click **⋮ >  Your Node > Ksmtuned tab.**
+
+![View Ksmtuned Statistics](/img/v1.1/host/view-ksmtuned-statistics.png)
+
+### **Parameters**
+
+**Run Strategy：**
+
+- **Stop:** Stop Ksmtuned and KSM，VM can still use shared memory pages.
+- **Run:** Run Ksmtuned。
+- **Prune:** Stop Ksmtuned and prune KSM memory pages.
+
+**Threshold Coefficient**:  configures the available memory percentage ratio, if the available memory is less than the threshold starts KSM, otherwise, it stops KSM.
+
+**Merge Across Nodes:**  specifies if pages from different NUMA nodes can be merged.
+
+**Mode:**
+
+- **Standard:** For the default mode, the control node ksmd uses a single CPU to about 20%, with the following parameters:
+
+```yaml
+Boost: 0
+Decay: 0
+Maximum Pages: 100
+Minimum Pages: 100
+Sleep Time: 20
+```
+
+- **High-performance:** Node ksmd uses from 20% to 100% of the single CPU and has faster scanning and merging efficiency, with the following parameters:
+
+```yaml
+Boost: 200
+Decay: 50
+Maximum Pages: 100
+Minimum Pages: 10000
+Sleep Time: 20
+```
+
+- **Customized:** You can customize the configuration to reach the performance that you want.
+
+Ksmtuned uses the following parameters to control KSM efficiency by configuring:
+
+- **Boost**: The number of scanned pages is incremented each time, and if the available memory is less than the **Threshold Coefficient**.
+- **Decay**: The number of scanned pages is decremented each time, and if the available memory is greater than the **Threshold Coefficient**.
+- **Maximum Pages**: Maximum number of pages per scan.
+- **Minimum Pages**: The minimum number of pages per scan, also the configuration for the first run.
+- **Sleep Time(ms)**: The interval between each detection, is calculated by the formula (**Sleep Time** \* 16 \* 1024 \* 1024 / Total Memory) for the interval between two scans, at least 10ms.
+
+**Here is a 512GiB memory node, assume the following parameters:**
+
+```yaml
+Boost: 300
+Decay: 100
+Minimum Pages: 1000
+Maximum Pages: 5000
+Sleep Time: 50
+```
+
+When Ksmtuned starts, initialize `pages_to_scan` in KSM to 1000 (**Minimum Pages**) and set `sleep_millisecs` to 10 (50 \* 16 \* 1024 \* 1024 / 536870912 KiB < 10).
+
+Start running the KSM when the available memory is below the **Threshold Coefficient**, and if it is detected as running every period (1 minute), `pages_to_scan` is incremented by 300 (**Boost**) until 5000 (**Maximum Pages**).
+
+Stop running KSM when the available memory is higher than the **Threshold Coefficient**, and decrement `pages_to_scan` by 100 (**Decay**) until 1000 (**Minimum Pages**) if it detects that it has stopped each period.
```

Comment:
```suggestion
KSM is started when the available memory is below the **Threshold Coefficient**, and if it is detected as running in each period (1 minute), `pages_to_scan` is incremented by 300 (**Boost**) until it reaches 5000 (**Maximum Pages**).

KSM is stopped when the available memory is higher than the **Threshold Coefficient**, and if it is detected as stopped in each period, `pages_to_scan` is decremented by 100 (**Decay**) until it reaches 1000 (**Minimum Pages**).
```

Reply:
We can use the same structure for these two sentences.
I changed these sentences according to my understanding. Make sure the meaning is still correct before you accept it.